From 9febf23ae39c6b7a49b3e9a43319ec9dca8ea771 Mon Sep 17 00:00:00 2001
From: Krzysztof Sywula <krzysztof.m.sywula@intel.com>
Date: Wed, 29 Jan 2014 17:32:13 +0000
Subject: [PATCH] Regenerating Khem's patch for assembler-arm.cc

deps/v8/src/platform-linux.cc is already fixed in this version of nodejs,
so no need to patch it again.
---
 deps/v8/src/arm/assembler-arm.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deps/v8/src/arm/assembler-arm.cc b/deps/v8/src/arm/assembler-arm.cc
index 9be62a4..f919ff1 100644
--- a/deps/v8/src/arm/assembler-arm.cc
+++ b/deps/v8/src/arm/assembler-arm.cc
@@ -71,10 +71,10 @@ static unsigned CpuFeaturesImpliedByCompiler() {
   // If the compiler is allowed to use VFP then we can use VFP too in our code
   // generation even when generating snapshots. ARMv7 and hardware floating
   // point support implies VFPv3, see ARM DDI 0406B, page A1-6.
-#if defined(CAN_USE_ARMV7_INSTRUCTIONS) && defined(__VFP_FP__) \
+#if defined(CAN_USE_ARMV7_INSTRUCTIONS) && defined(__ARM_PCS_VFP) \
     && !defined(__SOFTFP__)
   answer |= 1u << VFP3 | 1u << ARMv7 | 1u << VFP2;
-#endif  // defined(CAN_USE_ARMV7_INSTRUCTIONS) && defined(__VFP_FP__)
+#endif  // defined(CAN_USE_ARMV7_INSTRUCTIONS) && defined(__ARM_PCS_VFP)
         // && !defined(__SOFTFP__)
 #endif  // _arm__
   if (answer & (1u << ARMv7)) {
-- 
1.7.12.2

